src/css/components/_alert.scss

```diff
@@ -102,6 +102,10 @@
     padding: 15px;
 }
 
+.alert-buttons .soft-button-image-static {
```

Comment:
```suggestion
.alert-buttons .soft-button-image, .soft-button-image-static {
```
Issue also applies to softbuttons with dynamic images
 
![Screenshot from 2021-05-18 11-58-54](https://user-images.githubusercontent.com/10703941/118696185-05a01f00-b7dc-11eb-82f4-9e39eb9e4c95.png)

Reply:
Addressed in https://github.com/smartdevicelink/generic_hmi/pull/388/commits/6f869b98f9f3559e31129e4a955b4b48e2d040c0